docs/src/pages/components/menu/demo.tsx

```diff
@@ -1,37 +1,61 @@
 import * as React from 'react';
+import { Divider, Menu, MenuItem } from '@aws-amplify/ui-react';
 
-import { Divider, Menu, MenuItem, View } from '@aws-amplify/ui-react';
-import { MenuPropControls } from '@/components/MenuPropControls';
-import { useMenuProps } from '@/components/useMenuProps';
-import { Example } from '@/components/Example';
+import { Demo } from '@/components/Demo';
+import { MenuPropControls, MenuPropControlsProps } from './MenuPropControls';
+import { useMenuProps } from './useMenuProps';
+
+const propsToCode = (props: MenuPropControlsProps) => {
+  const size = props.size ? `\n  size=${props.size}` : '';
+  return `<Menu ${size}
+  menuAlign={${props.menuAlign}}
+  >
+      <MenuItem onClick={() => alert('Download')}>
+        Download
+      </MenuItem>
+      <MenuItem onClick={() => alert('Create a Copy')}>
+        Create a Copy
+      </MenuItem>
+      <MenuItem onClick={() => alert('Mark as Draft')}>
+        Mark as Draft
+      </MenuItem>
+      <Divider />
+      <MenuItem isDisabled onClick={() => alert('Delete')}>
+        Delete
+      </MenuItem>
+      <MenuItem onClick={() => alert('Attend a workshop')}>
+        Attend a workshop
+      </MenuItem>
+  </Menu>`;
+};
```

Comment:
nit: Can we adjust the indentation in the example above to match what it looks like after running prettier?

Reply:
Not sure I'm following the ask, should it look different?

Reply:
It looks like the closing `</Menu>` tag isn't vertically aligned with the opening `<Menu>` tag, and the rest appears to be indented with tabs rather than two spaces.

Reply:
gotcha, thanks, will update